mail-filter/afew/afew-3.0.1.ebuild

```diff
@@ -4,7 +4,8 @@
 EAPI=7
 
 DISTUTILS_SINGLE_IMPL=1
-PYTHON_COMPAT=( python{3_6,3_7} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{6,7,8} )
 
 inherit distutils-r1
```

Comment:
(Wanted to comment lower down but can't -- [change](https://dev.gentoo.org/~mgorny/python-guide/migration.html?highlight=python_usedep) PYTHON_MULTI_USEDEP to PYTHON_USEDEP, and sphinx should be a BDEPEND).